pkg/collector/cluster_test.go

```diff
@@ -104,7 +105,7 @@ func TestClusterCollectorGetFake(t *testing.T) {
 			discoveryClient := discoveryFake.FakeDiscovery{}
 			testOpts := ClusterOpts{ClientSet: clientset, DiscoveryClient: &discoveryClient}
 
-			collector, err := NewClusterCollector(&testOpts, []string{})
+			collector, err := NewClusterCollector(&testOpts, []string{}, []string{"kapp.k14s.io/original"})
```

Comment:
Let's not pass the additional `"kapp.k14s.io/original"` to every test run, but we can add it as another parameter to the `testCases` array, and only pass it with the relevant test.

Reply:
Yeah, that make sense :+1: